src/modules/AdminModule.cpp

```diff
@@ -721,7 +721,7 @@ void AdminModule::handleGetDeviceConnectionStatus(const meshtastic_MeshPacket &r
 #if HAS_BLUETOOTH
     conn.has_bluetooth = true;
     conn.bluetooth.pin = config.bluetooth.fixed_pin;
-#ifdef ARCH_ESP32
+#if defined(ARCH_ESP32) && !defined(CONFIG_IDF_TARGET_ESP32S2)
```

Comment:
Is this really neccessary? ESP32-S2 should not have HAS_BLUETOOTH defined.

Reply:
Updated as suggested.

Following updates were needed:
- change to `PaxCounterModule.cpp` is needed as the build will still pull in the cpp file regardless if the define is set
- change to `sleep.cpp` to exclude pulling in nimble if `MESHTASTIC_EXCLUDE_BLUETOOTH` is defined.

Reply:
Excellent - this way we can dodge the dreaded CONFIG_IDF macros ... :-)